### ISSUE TITLE
Possible that understat.com changed the class of a button?

### ISSUE THREAD

**Issue by aegonwolf** [oseymour/ScraperFC#32]
Hi me again, 

Is it possible Understat.com changed something on their website and Soccerdata is looking for a button that doesn't exist anymore?

I have this error when trying `homeaway = understat.scrape_home_away_tables(year=2024, league='EPL')` or any year. Worked fine 1 month ago.
```
selenium.common.exceptions.ElementClickInterceptedException: Message: element click intercepted: Element <button class="options-button">...</button> is not clickable at point (738, 554). Other element would receive the click: <label for="themes-switch" title="Light theme"></label>```

And I looked on the site, but did not actually find a button with that class.

Thanks for this great package!

**Comment by oseymour** [comment]
Yeah they must have moved something around. I'll take a look when I get a chance. Thanks!

**Comment by oseymour** [comment]
Fixed in v2.9.0
